[01;31m!! [D/T] Malformed group name ''hÃ©hÃ©hÃ©'', must match /[00;33m^[a-z]([-_.a-z0-9]*[a-z0-9][$]?)?$[0;0m[01;31m/i.[0;0m
>>> [01;31mException[0;0m: [01;36m<class 'licorn.foundations.exceptions.BadArgumentError'>[0;0m
>>> [01;32mCall trace[0;0m:
  File "/usr/lib/python2.6/dist-packages/licorn/interfaces/cli/__init__.py", line 47, in cli_main
    functions[mode][1](opts, args)
  File "../interfaces/cli/add.py", line 441, in add_group
    permissive = opts.permissive, force=opts.force)
  File "/usr/lib/python2.6/dist-packages/licorn/core/groups.py", line 275, in AddGroup
    styles.stylize(styles.ST_REGEX, hlstr.regex['group_name'])))

